db/seeds.rb

```diff
@@ -0,0 +1,7 @@
+{
+  admin: "Account with administrative rights",
+  moderator: "Moderator role. Allows to update other's posts and comments",
+  editor: "Account is allowed to create own posts, publish them and update them"
```

Comment:
I think for seeds it's not so important for now as they go to db actually. Further we could introduce  globalize, but not now